src/node_crypto.cc

```diff
@@ -652,6 +620,17 @@ void SecureContext::Init(const FunctionCallbackInfo<Value>& args) {
       min_version = TLS1_2_VERSION;
       max_version = TLS1_2_VERSION;
       method = TLS_client_method();
+    } else if (strcmp(*sslmethod, "TLSv1_3_method") == 0) {
```

Comment:
It's also not clear right now whether we actually need this for the QUIC impl at this point so I have no problem pulling this particular change back out (once I verify)

Reply:
Went ahead and pulled these out. If I do confirm they're needed for the QUIC impl, I'll add them through that PR.

Reply:
Sure, or open an issue and I can add (but I'd be surprised if QUIC depended on deprecated OpenSSL APIs, particularly since I'm pretty sure boringssl no-ops these specific APIs - they don't want apps configuring crypto algs via API).